src/renderer/modules/components/KeybindItem.css

```diff
@@ -0,0 +1,90 @@
+.rp-keybind-container {
```

Comment:
Speaking of how the keybind recorder looks, it needs to be as similar as possible to how Discord has it. Remember to use CSS variables.